geth/node/node.go

```diff
@@ -154,8 +156,10 @@ func activateEthService(stack *node.Node, config *params.NodeConfig) error {
 
 // activateShhService configures Whisper and adds it to the given node.
 func activateShhService(stack *node.Node, config *params.NodeConfig) error {
+	logger := log.New("package", "geth/node")
```

Comment:
maybe extract as a global variable here?
or, alternatively, provide a symbol name too: `log.New("package", "geth/node.activateShhService")`

Reply:
Before I refactor: would we want all loggers that *aren't* struct fields to just be initialized in package scope rather than declaring them locally while needed? 

Upsides:
* A single logging instance to call into
* You no longer have to worry about initializing a logging instance each and every time you want to log something in a method that doesn't have a receiver

Downsides:
* (slightly) higher memory usage
* If we did a per function logger, we could get per-function log granularity by doing something similar to what you suggested by appending  `.<function name>` to the end of the package value

Reply:
no, I want them as struct fields, but on functions, I'd like to have a global variable.

Reply:
Right - my question may have been unclear, but that was what I was getting at. I think that makes sense. I'll add a global logger to each package to cut down on the per-function logging clutter